docs/docs/build/presence.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Presence
```

Comment:
You'll also want to specify `sidebar_position: <number>` to control the ordering for how this appears in the sidebar.